app/routes/session.py

```diff
@@ -98,13 +98,16 @@ def login():
 
 
 def validate_jti(decrypted_token):
-    expires = datetime.utcfromtimestamp(decrypted_token["exp"]).replace(tzinfo=tzutc())
-    if expires < datetime.now(tz=tzutc()):
+    expires_at = datetime.utcfromtimestamp(decrypted_token["exp"]).replace(
+        tzinfo=tzutc()
+    )
+    jwt_expired = expires_at < datetime.now(tz=tzutc())
+    if jwt_expired:
         raise Unauthorized
```

Comment:
```suggestion
    if expires_at < datetime.now(tz=tzutc()):
        raise Unauthorized
```
jwt_expired is only used here and doesn't really add any context that isn't obvious from the boolean comparison

Reply:
As discussed, will keep as is for now for readability/consistency sakes.